src/requests.jl

```diff
@@ -263,7 +263,7 @@ See also [`request_json`](@ref) and [`request_raw`](@ref).
     query::Union{AbstractDict,Nothing} = nothing,
     require_base_url::Symbol = :strict,
     kwargs...,
-) where {T}
+)::T where {T}
```

Comment:
In my experience it is better and safer to specify the return type than to just convert the result of the parser:
```julia
julia> d = Dict{String,Any}("a" => 1 // 2)
Dict{String, Any} with 1 entry:
  "a" => 1//2

julia> f(x)::Float64 = x
f (generic function with 1 method)

julia> g(x) = x::Float64
g (generic function with 1 method)

julia> @inferred(f(d["a"]::Real))
0.5

julia> @inferred(g(d["a"]::Real))
ERROR: TypeError: in typeassert, expected Float64, got a value of type Rational{Int64}
Stacktrace:
 [1] g(x::Rational{Int64})
   @ Main ./REPL[6]:1
 [2] top-level scope
   @ REPL[8]:1
```

Reply:
Cool, yeah I'm fine with restoring the `::T` here.